packages/core/src/browser/cookie.ts

```diff
@@ -1,29 +1,25 @@
 import { display } from '../tools/display'
 import { ONE_MINUTE, ONE_SECOND } from '../tools/utils/timeUtils'
-import { findCommaSeparatedValue, generateUUID } from '../tools/utils/stringUtils'
+import { generateUUID } from '../tools/utils/stringUtils'
```

Comment:
This is the actual change of the library.
We are replacing cookies with a local map, so that we are GDPR conform.